packages/api/README.md

```diff
@@ -4,7 +4,7 @@ The Polkadot-JS API provides easy-to-use wrappers around JSONRPC calls that flow
 
 The API wrappers provide a standard interface for use -
 
-- A static `.create(<optional WsProvider>)` that returns an API instance when connected, decorated and ready-to use
+- A static `.create(<optional WsProvider>, <optional CustomTypes>)` that returns an API instance when connected, decorated and ready-to use
```

Comment:
This should be `.create({ provider:<optional WsProvider>, types: <optional CustomTypes>})`when both parameters are passed...